collage/layout.py

```diff
@@ -0,0 +1,213 @@
+import tkinter as tk
+from tkinter.colorchooser import askcolor
+from matplotlib import font_manager
+import numpy as np
+from datetime import datetime
+
+def get_system_fonts():
+    fonts = []
+    for x in font_manager.findSystemFonts():
+        x = x[::-1]
+        dot = x.find('.')
+        slash = x.find('\\')
+        x = x[slash-1:dot:-1]
+        fonts += [x]
+    return np.unique(fonts)  
+
+
+class Application(tk.Frame):
+    def __init__(self, master=None):
+        super().__init__(master)
+        self.master = master
+        self.grid()
+        self.create_widgets()
+
+    def create_widgets(self):
```

Comment:
Add additional methods to initialise some widgets. This method is too big, and I think it should be separated.